dist/index.js

```diff
@@ -293,3 +293,9 @@ module.exports = {
   }
 };
 
+var OutboundLink = require('./components/OutboundLink');
+OutboundLink.outboundTrackingFunction = reactGA.outboundLink;
```

Comment:
weird i thought i updated the name through a follow-up commit yesterday :unamused: